nodebuilder/header/service.go

```diff
@@ -0,0 +1,73 @@
+package header
+
+import (
+	"context"
+
+	"github.com/celestiaorg/celestia-node/header"
+	"github.com/celestiaorg/celestia-node/header/p2p"
+	"github.com/celestiaorg/celestia-node/header/sync"
+)
+
+type Service interface {
+	// Start starts the header service.
+	Start(context.Context) error
+	// Stop stops the header service.
+	Stop(context.Context) error
```

Comment:
Do we need to keep those Start and Stop in each service by the way? I would remove those and make a separate API or Service in the node module that can granularly start and stop a service/module by it's name.

Reply:
Right now implementations are just empty functions. They were there before, but it's time to remove those.

Reply:
done